Feature add css

1devNdogs opened the pull request:
# Feature add css

Mainly add minified css with custom styles and assets folder with chains logos.


## Summary
- Add a custom bootstrap 4 css in `./src/assets/css` 
- Add `./src/assets/img/*` folder with all parachain and subspace logos.
- Update `package.json` with needed dependencies and move @types to devDependencies
- Update `tsconfig.json` update `"allowJs": false,`
- Update `.gitignore` added build folder
- Add css import on `./src/index.tsx`
- Update public/index.html adding bootstrap and updated meta and page title.

isSerge commented on package.json:
```diff
@@ -3,17 +3,23 @@
-    "bootstrap": "^5.1.1",
+    "bootstrap": "4.6.0",
+    "classnames": "2.3.1",
+    "moment": "2.29.1",
```
have u considered using `date-fns` instead of `moment`? it's much smaller

1devNdogs replied:
Oh, i will check that, if its ligther better, as I'am just using a date formatter in the one row label, will consider it for next changes.

nazar-pc commented on .gitignore:
```diff
@@ -1 +1,2 @@
 /node_modules
+/build
```
New line in all text files, please. Just configure your IDE to add it automatically

nazar-pc reviewed (commented):
Can't CSS file be loaded from a dependency instead of copy-pasting it into the codebase?

1devNdogs commented:
About:
> Can't CSS file be loaded from a dependency instead of copy-pasting it into the codebase?

I exported that minified css in the previous [commit](https://github.com/subspace/subspace-relayer-front-end/blob/2cf3b8a1781a7c4b363f524838ead93f3c2da595/package.json#L31-L35) It contains extended bootstrap styles, trying to match Subspace styles. 
I decided to remove all that generation scripts and dependencies and for now import it this way on this initial version.
I'am having a call with Diana tomorrow, getting the final styles I was thinking we could discuss how to handle the Subspace Design System in somethink like a package. So that manual import of the minified css should be temporal.

nazar-pc commented:
I do not see any documentation about where that file comes from and how would I generate it myself. This is critical for project maintenance.

1devNdogs commented:
I added build and runing instructions for this branch, it also add a lot of scss files for the source build. Also updated the PR description.
Full running instructions are included in the [dashboard](https://github.com/subspace/subspace-relayer-front-end/tree/feature-dashboard-relayer) branch.
